src/wagtailtrans/migrations/0001_initial.py

```diff
@@ -32,9 +32,9 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='TranslatablePage',
             fields=[
-                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
+                ('translatable_page_ptr', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, related_name='+', serialize=False, to='wagtailcore.Page')),
```

Comment:
I tried out the PR on an existing project, doing a `makemigration` Django expects to give an initial value to `translatable_page_ptr`, while giving the default value as `1` , the migration error outs with a `sqlite3.IntegrityError: NOT NULL constraint failed: home_homepage__new.page_ptr_id`

<img width="850" alt="screen shot 2016-11-09 at 6 46 36 pm" src="https://cloud.githubusercontent.com/assets/1276650/20139690/59ead676-a6ad-11e6-978e-9f59bd01326c.png">